src/ui/public/courier/fetch/request/doc.js

```diff
@@ -1,17 +1,14 @@
-import DocStrategyProvider from '../strategy/doc';
 import AbstractRequestProvider from './request';
 
 export default function DocRequestProvider(Private) {
 
-  const docStrategy = Private(DocStrategyProvider);
   const AbstractRequest = Private(AbstractRequestProvider);
 
   class DocRequest extends AbstractRequest {
```

Comment:
Now that this is referred to as `AbstractDocRequest` we should rename it here, and in the file name too